tests/client/amo/components/TestOverallRating.js

```diff
@@ -132,22 +141,33 @@ describe('OverallRating', () => {
         });
 
         mockApi
-          .expects('postRating')
+          .expects('submitReview')
           .withArgs(params)
           .returns(Promise.resolve(ratingResponse));
 
-        return actions.createRating({ ...params, userId }).then(() => {
-          assert.equal(dispatch.called, true);
-          const action = dispatch.firstCall.args[0];
-
-          assert.equal(action.type, SET_REVIEW);
-          assert.equal(action.data.addonId, params.addonId);
-          assert.equal(action.data.userId, userId);
-          assert.deepEqual(action.data.rating, ratingResponse.rating);
-          assert.deepEqual(action.data.versionId, ratingResponse.version.id);
-
-          mockApi.verify();
-        });
+        return actions.createRating(
+          {
```

Comment:
We use two spaces everywhere to show block continuations. The thing I like about this style is that if your eye is trained to count two spaces as a continuation then this is unambiguous:
````
return actions.createRating(
  {
    foo,
  }).
  .then(() => ...);
````
This approach introduces a four space indent which throws me off personally:
````
return actions.createRating({
    foo,
  }).
  .then(() => ...);
````
I know it's just a nit but whatever we decide it should be consistent throughout (two spaces or four or mixed spaces with well defined execptions)